content/blog/2020/08/camel-quarkus-release-1.0.0/index.md

```diff
@@ -0,0 +1,77 @@
+but also for [command-style applications](https://camel.apache.org/blog/2020/07/command-line-utility-with-camel-quarkus/)
+that execute some task and exit.
+
+## Developer joy
+
+Writing integrations has never offered more developer joy than with Camel Quarkus:
+
+* Easy to start with: either via [code.quarkus.io](https://code.quarkus.io/) or using `mvn quarkus:create`
+* The "live coding" feature, a.k.a. `mvn compile quarkus:dev` shortens your development iterations
+* Easy to test using JUnit 5 extensions
+* Shields from the complexity of the native compilation using [GraalVM](https://www.graalvm.org/) or
+  [Mandrell](https://github.com/graalvm/mandrel)
+
+
+## The journey to 1.0.0
+
+[Luca](https://github.com/lburgazzoli) and [Guillaume](https://github.com/gnodet) started porting Camel to
+[Quarkus](https://github.com/quarkusio/quarkus) very early while Quarkus was still in the stealth mode at Red Hat.
+The work started around November 2018 and was
+[merged to Quarkus master](https://github.com/quarkusio/quarkus/commit/549c74d74f4ae6af83854e51f14db429adef7483) on Feb.
+28, 2019 - shortly before Quarkus was
+[publicly announced](https://developers.redhat.com/blog/2019/03/07/quarkus-next-generation-kubernetes-native-java-framework/)
+on March 7, 2019. At that time, Quarkus
+[supported just a handful of Camel components](https://github.com/quarkusio/quarkus/tree/0.11.0/extensions/camel)
+
+The next important milestone was donating the Camel specific bits from the main Quarkus source tree to Apache Software
+Foundation (ASF). This happened around July 2019 and the current
+[GitHub home of the project](https://github.com/apache/camel-quarkus) is the result.
+
+Under the new home, the project attracted a fair amount of new contributors which in turn brought a lot of
+contributions.
+
+## Where are we now?
+
+Since the early beginnings the number of supported Camel bits went up substantially. These are the numbers of Camel
+components supported on Quarkus as of Quarkus 1.0.0 and Camel 3.4:
+
+* 155 out of 350 Camel components (44%)
+* 27 out of 43 Camel data formats (63%)
+* 13 out of 17 Camel languages (76%)
+* 17 out of 42 miscellaneous Camel components (40%)
+
+The complete list is [here](https://camel.apache.org/camel-quarkus/latest/list-of-camel-quarkus-extensions.html)
+
+## Who is using Camel Quarkus?
+
+While we mostly have only indirect information about the usage via
+[GitHub issues](https://github.com/apache/camel-quarkus/issues), there is one prominent user, we are especially proud
+of: [Camel K](https://camel.apache.org/camel-k/latest/index.html). As of
+[Camel K 1.0](https://camel.apache.org/blog/2020/06/camel-k-release-1.0.0#fast-startup-and-low-memory), Camel Quarkus is
+one of the two supported runtimes and is "is expected to be the default underlying runtime in the next release".
+
+## Highlights of the release 1.0.0
+
+TODO:
+bootstrapping methods
+reflectionless
```

Comment:
> reflectionless

We can't really make that claim yet as https://github.com/apache/camel-quarkus/issues/1108 wont be fixed until we get an updated Camel release.

Reply:
Indeed, let me remove it.